code/html/custom.js

```diff
@@ -60,7 +60,10 @@ function magnitudeType(type) {
         "Current", "Voltage", "Active Power", "Apparent Power",
         "Reactive Power", "Power Factor", "Energy", "Energy (delta)",
         "Analog", "Digital", "Event",
-        "PM1.0", "PM2.5", "PM10", "CO2", "Lux", "UV", "Distance" , "HCHO",
+        "PM1.0", "PM2.5", "PM10",
+        "Particles ≧0.3 ﹤0.5µm", "Particles ≧0.5 ﹤1µm", "Particles ≧1 ﹤2.5µm ",
+        "Particles ≧2.5 ﹤5µm", "Particles ≧5 ﹤10µm", "Particles ≧10µm",
```

Comment:
According to the data sheet:

![screen shot 2018-08-09 at 01 54 17](https://user-images.githubusercontent.com/288709/43871982-2b5ad4c6-9b77-11e8-9b03-a2b184b57f37.png)

I don't really understand how the device can measure "beyond 10µm" if that it's max range. What am I missing?